package.json

```diff
@@ -100,5 +105,8 @@
     "terra-dev-site": "^2.0.0",
     "terra-toolkit": "^4.5.0",
     "webpack-merge": "^4.1.2"
+  },
+  "dependencies": {
```

Comment:
This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.